src/components/reservations/ReservationDetail.js

```diff
@@ -0,0 +1,193 @@
+`;
+
+const Content = styled(DialogContent)`
+  padding: 0.5rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  & > .desc {
+    font-size: 1rem;
+    font-weight: bold;
+    font-family: 'Urbanist', sans-serif;
+  }`;
+
+const Dates = styled(Typography)`
+display: flex;
+flex: 1;
+flex-direction: row;
+justify-content: space-between;
+width: 100%;
+margin-bottom: 1rem;
+`;
+
+const Date = styled(Typography)`
+  display: flex;
+  padding-left: 1rem;
+  font-size: 1rem !important;
+`;
+
+const DateContent = styled(Typography)`
+    font-size: 1rem;
+    padding-left: 0.5rem;
+    border-bottom: 1px double #ccc;
+`;
+
+const AmountText = styled(Typography)`
+  font-size: 1rem;
+  text-shadow: 1px 1px 1px #5688ae;
+  font-family: 'Urbanist', sans-serif;
+  color: black;
+  border-bottom: 1px solid #ccc;
+`;
+
+const BootstrapDialog = styled(Dialog)(({ theme }) => ({
+  '& .MuiDialogContent-root': {
+    padding: theme.spacing(3),
+    boxShadow: '0px 0px 10px 0px rgba(0,0,0,0.75)',
+  },
+  '& .MuiDialogActions-root': {
+    padding: theme.spacing(2),
+    boxShadow: '0px 0px 10px 0px rgba(0,0,0,0.75)',
+  },
+}));
+
+const BootstrapDialogTitle = (props) => {
+  const { children, onClose, ...other } = props;
+
+  return (
+    <Title sx={{ m: 0, p: 2 }} {...other}>
+      {children}
+      {onClose ? (
+        <IconButton
+          aria-label="close"
+          onClick={onClose}
+          sx={{
+            position: 'absolute',
+            right: 8,
+            top: 8,
+            color: (theme) => theme.palette.grey[500],
+          }}
+        >
+          <CloseIcon />
+        </IconButton>
+      ) : null}
+    </Title>
+  );
+};
+
+BootstrapDialogTitle.propTypes = {
+  children: PropTypes.node.isRequired,
+  onClose: PropTypes.func.isRequired,
+};
+
+const ReservationDetail = ({ reservationDetail }) => {
+  const { start_date, end_date, car } = reservationDetail;
+  const dailyPrice = car.description.price_daily;
+
+  const [open, setOpen] = useState(false);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <>
+      <DetailBtn variant="outlined" onClick={handleClickOpen}>
+        Reservation Detail
+      </DetailBtn>
+
+      <BootstrapDialog
+        onClose={handleClose}
+        aria-labelledby="customized-dialog-title"
+        open={open}
+      >
+        <BootstrapDialogTitle id="customized-dialog-title" onClose={handleClose}>
+          Reservation Detail
+        </BootstrapDialogTitle>
+        <Content dividers>
+          <Dates>
+            <Date>
+              <DateRange />
+              <DateContent>
+                {start_date}
+              </DateContent>
+            </Date>
+            <Date>
+              <DateRange />
+              <DateContent>
+                {end_date}
+              </DateContent>
+            </Date>
+          </Dates>
```

Comment:
- [ ] I would add a validation here, because I could reserve a car from 02/20 to 02/04, then I had -16 days to return the car and you owned me some money, which can't happen in the real world.

Reply:
We are working on this issue, Thank you for your feedback @MuriloRoque 🙏🏼

Reply:
@MuriloRoque Problem solved :raised_hands: